subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -358,6 +358,11 @@ Generate a build scan with detailed performance diagnostics.
 
 image::gradle-core-test-build-scan-performance.png[Build Scan performance report]
 
+`--watch-fs`, `--no-watch-fs`::
+Toggles <<gradle_daemon.adoc#sec:daemon_watch_fs,watching the file system>>.
```

Comment:
We should consistently use either `file system` or `file-system`. I think we use `file-system` more often, so maybe standardize on that?

Reply:
Using `file-system` now consistently. See https://github.com/gradle/gradle/pull/13219.